backend/chromeApiServer/src/main/java/com/canvas/chromeApiServer/ChromeApiController.java

```diff
@@ -82,6 +85,31 @@ public ResponseEntity<CommandOutput> compileCodeFile(
         return new ResponseEntity<>(commandOutput, HttpStatus.OK);
     }
 
+    /**
+     * Test Route to get student submission given access token and studentId
+     * Submission is expected to be saved under my files/CanvasCode/sample.cpp
+     * @param studentId
+     * @param token
+     * @return
+     */
+
+        @GetMapping(
+            value = "/fetchStudentSubmission",
+            produces = { "application/json" }
+    )
+
+    public ResponseEntity<String> fetchFileFromCanvasAndSave(@RequestParam("studentId") String studentId,
```

Comment:
why do we need an API to fetch the student submission? I thought the `/evaluate` API would take in the submission files as input from the frontend. I don't think the student submission would be stored anywhere in Canvas yet either before `/evaluate` would be called

Reply:
This is a test route as mentioned in the comments. we do not need this. This is to test fetching a file from canvas for the instructor. We would receive the student id from the front-end and then we fetch the submission file.